meinberlin/apps/votes/models.py

```diff
@@ -67,6 +71,48 @@ def project(self):
     def is_valid_for_item(self, item):
         return item.module == self.module
 
+    @staticmethod
+    def get_hash_for_token(token, module):
```

Comment:
The naming is already very on point and understandable and total nitpick, but to me the `get` is for stuff that already exists, so this could be `create_hash_for_token` or just `hash_token`?